pkg/fetch/vendir_test.go

```diff
@@ -58,32 +58,52 @@ func Test_AddDir_skipsTLS(t *testing.T) {
 	}
 }
 
-func TestExtractImageRegistry(t *testing.T) {
```

Comment:
Let's also add a test similar to the one above (Test_AddDir_skipTLS) for git urls.